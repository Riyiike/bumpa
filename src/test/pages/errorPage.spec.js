import React from "react";
import { shallow } from 'enzyme'
import ErrorPage from "../../pages/errorPage";

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useRouteError: () => [],

}))



describe('test mainPage', () => {
    it('test mainPage', () => {
        const btn = shallow(<ErrorPage />)
        expect(btn).toMatchSnapshot();
    })
})