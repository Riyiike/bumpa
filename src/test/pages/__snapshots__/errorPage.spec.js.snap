// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test mainPage test mainPage 1`] = `
<div
  className=""
  id="error-page"
>
  <h1>
    Oops!
  </h1>
  <p>
    Sorry, an unexpected error has occurred.
  </p>
  <p>
    <i />
  </p>
</div>
`;
